chef/cookbooks/rabbitmq/files/suse/epmd.socket-listen.conf

```diff
@@ -0,0 +1,3 @@
+[Socket]
+ListenStream=
```

Comment:
Is this intended? First an empty value and afterwards some other value for the same variable?